types/artifacts.d.ts

```diff
@@ -169,6 +171,14 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      /** @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#Attributes */
+      export interface LinkElement {
+        rel: string
+        href: string
+        as: string
+        crossOrigin: string|null
```

Comment:
not worth the extra specificity?

Reply:
`'anonymous'|'use-credentials'|null` instead of just `string|null`. It might not buy us anything (maybe if we end up with a switch statement somewhere).

Reply:
oh gotcha, sure!